composer.json

```diff
@@ -59,7 +59,7 @@
         "daverandom/libdns": "^2.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.8.31|^5.7.3",
+        "phpspec/prophecy": "^1.8",
```

Comment:
Do we still need this?

Reply:
it's used in tests (and not installed by simple-phpunit)